scripts/tunnel-creation.mjs

```diff
@@ -0,0 +1,351 @@
+#!/usr/bin/env node
+/**
+ * Test script for creating lockdown service, starting CoreDeviceProxy, and creating tunnel
+ * This script demonstrates the tunnel creation workflow for all connected devices
+ */
+import {logger} from '@appium/support';
+import _ from 'lodash';
+import {
+  PacketStreamServer,
+  TunnelManager,
+  createLockdownServiceByUDID,
+  createUsbmux,
+  startCoreDeviceProxy,
+  startTunnelRegistryServer,
+} from 'appium-ios-remotexpc';
```

Comment:
we must explicitly ignore linter error if we want to keep this script in the repository and leave a TODO to remove it as soon as the package is published